src/coreclr/System.Private.CoreLib/src/System/Runtime/RuntimeImports.CoreCLR.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Runtime
+{
+    internal static class RuntimeImports
```

Comment:
Can these definitions now be shared with nativeaot?

Reply:
Sure - I moved it to the common location.

Reply:
I might be missing something, but it looks like not shared with nativeaot